routes/user.js

```diff
@@ -22,9 +22,10 @@ const urlEncodedParser = bodyParser.urlencoded({ extended: false });
 
 router.post("/add", urlEncodedParser, async (req, res) => {
   const email = req.body.email;
-  const verificationToken = await DB.addSubscriberUnverifiedEmailHash(email);
+  const unverifiedSubscriber = await DB.addSubscriberUnverifiedEmailHash(email);
+  const verificationToken = unverifiedSubscriber.verification_token;
 
-  const url = `${AppConstants.SERVER_URL}/user/verify?token=${encodeURIComponent(verificationToken)}&email=${encodeURIComponent(email)}`;
+  const url = `${AppConstants.SERVER_URL}/user/verify?token=${encodeURIComponent(verificationToken)}`;
```

Comment:
I decided to drop the extra use of the plaintext email in the verification link to prevent leakage via any `Referer` values.